### PR TITLE
Fix parsing + some code refactoring

### PR DESCRIPTION
**Pull request by ilyankou**

### CODE REVIEW

**Comment by ilyankou** [comment]
I removed a rule from `lumper.js` that turns `Value + Noun` into a `Value`. That is, previously `5 books` was marked as a Value, now it is `['Value', 'Noun']`. I also added two rules to `grammar_rules.js` that turn `Value + Verb` into `Value + Noun` ('5 kittens'), and `Adjective + Verb` into `Adjective + Noun` ('red roses').

Fixes #205, #198, #194, #193

File: test/unit_tests/sentence/tagger.js [hunk 1]
```diff
@@ -65,3 +66,6 @@ describe('custom lexicon', function() {
   });
 
 });
+
+console.log(nlp.text('3 buses').terms());
```
Comment (spencermountain) [inline test/unit_tests/sentence/tagger.js]: heads up ☝️

**Comment by spencermountain** [comment]
yeah, thanks ilya, if it made it easier, you could also coerce '5 trains' into a `Value`, although I like what you've done.
just comment those console.logs and we're good-to-go

**Comment by ilyankou** [comment]
Fixed!

**Comment by spencermountain** [comment]
hey, just pushing a build and wanted to check something with you
[this `w` variable is redeclared](https://github.com/nlp-compromise/nlp_compromise/blob/master/src/term/noun/value/value.js#L251), 
can i just rename it to something else?
(caught it in the linter)

**Comment by ilyankou** [comment]
Sure go ahead!

**Pull request merged**